programs/squads_multisig_program/src/instructions/proposal_vote.rs

```diff
@@ -119,6 +146,98 @@ impl ProposalVote<'_> {
     }
 }
 
+impl ProposalVoteV2<'_> {
```

Comment:
@vovacodes @bguillaumat alot of this seems redundant. Should we limit to this specific cancel and throw panic on the others? or keep it and eventually move everything through ProposalVoteV2? Seems like overkill for just one more system account.